Marquee/ViewControllers/DetailViewController.swift

```diff
@@ -19,22 +19,19 @@ class DetailViewController: UIViewController {
     @IBOutlet weak var releaseDate: UILabel!
     @IBOutlet weak var contentOverview: UILabel!
     @IBOutlet weak var contentTitle: UILabel!
-    var selectedContent = Content(overview: "", poster_path: "", release_date: "", title: "", vote_average: 0.0)
-    let favoritesViewController = FavoritesViewController()
+    var selectedContent = Content(overview: "", posterPath: "", releaseDate: "", title: "", voteAverage: 0.0)
     let viewController = ViewController()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        userScore.text = "Rating: \(selectedContent.vote_average)"
-        scoreBar.progress = Float(selectedContent.vote_average*0.1)
+        userScore.text = "Rating: \(selectedContent.voteAverage)"
+        scoreBar.progress = Float(selectedContent.voteAverage*0.1)
         contentTitle.text = selectedContent.title
         contentOverview.text = selectedContent.overview
-        releaseDate.text = selectedContent.release_date
-        
-        let url = URL(string: "https://image.tmdb.org/t/p/w500/\(selectedContent.poster_path)")
+        releaseDate.text = selectedContent.releaseDate
+        let url = URL(string: "https://image.tmdb.org/t/p/w500/\(selectedContent.posterPath)")
```

Comment:
Might be a good idea to keep all of your raw strings in a Constant struct. I left a similar comment in one of your other PRs.